src/primitives.jl

```diff
@@ -1,18 +1,31 @@
+const THREADED_KEYWORD = "- `threaded`: `true` or `false`. Whether to use multithreading. Defaults to `false`."
```

Comment:
Oh this is such a good way of not having to retype the same documentation over and over. I am so stealing this for my other project 😆

Reply:
Yeah, from Rasters.jl experience. Actually the main reason is so users see exactly the same things everywhere and it feels consistent